MetaMask/metamask-extension
# Bump controllers to ^27.0.0

## Explanation



The new version of `controllers` includes an important change which
greatly relieves pressure on Infura when the Gas API goes down.

## More information



* https://github.com/MetaMask/controllers/pull/712
* https://github.com/MetaMask/metamask-extension/pull/13865
* https://consensys.slack.com/archives/GV90KPED9/p1646339312789189 (and other related issues in that channel)

## Manual testing steps



- Open MetaMask (popup or full-screen view).
- Go to Settings and turn on the Enhanced Gas Fee UI setting.
- Go to send a transaction.
- Proceed to the confirm transaction screen.
- Click on "Market" (or whatever priority level is currently selected).
- Note that all of the data under Network Status should be present.
- Click on "Advanced".
- Note that all of the information under each field should be present ("current" and "12hr").
- Close the Advanced modal.
- Find the extension in your browser and open up the inspector for background.html.
- Go to the Network tab.
- Find a request that looks like `suggestedGasFees`. Right-click on it and choose "Block request URL".
- Open the MetaMask popup again.
- Note that 2 out of the 3 sections in Network Status should be gone (only Base Fee should be there).
- Open Advanced.
- Note that most of the information under each field should be gone (only "current" should be there).
